Added test project for Android embedding.

matthew-carroll opened the pull request:
The UI deployed by the app is currently just a vertical list of buttons that will launch various Flutter experiences.

There is a single Espresso test suite with a single Espresso test that verifies the first button is visible.

There is a build_and_test.sh which builds the app and runs the espresso test.  There is no logic currently for parsing the test output and verifying success.

Commit:
Added test project for Android embedding.

Commit:
Added test project for Android embedding.

dnfield commented on dev/integration_tests/android_embedding/.metadata:
```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
```
Are the two files above this supposed to be checked in? last-build.bin and gc.properties?

matthew-carroll replied:
I have no idea.  I don't know what those files are, but it would seem that they weren't covered by the standard .gitignore...

dnfield commented on dev/integration_tests/android_embedding/README.md:
```diff
@@ -0,0 +1,16 @@
+# android_embedding
```
Can we update this with desriptive narrative of what this folder is for?  Perhaps just your message in the PR comment?

matthew-carroll replied:
Done

dnfield commented on dev/integration_tests/android_embedding/pubspec.yaml:
```diff
@@ -0,0 +1,72 @@
+name: android_embedding
+description: A new Flutter project.
+
```
add `publish_to: none`

dnfield replied:
I'd also generally just remove the comments/unnecessary bits from this file if possible.

matthew-carroll replied:
Done

Commit:
PR Updates

Commit:
PR Updates

dnfield commented:
LGTM

Commit:
WIP

Commit:
WIP

Commit:
Commit to repro issue

Commit:
Commit to repro issue

Commit:
WIP

Commit:
WIP

Commit:
WIP

Commit:
WIP

Commit:
WIP

Commit:
Add null checks to coverage collection (#31092)

engine-flutter-autoroll committed:
0575a9b46 Roll src/third_party/skia bf15b6676843..d51d00745b1d (6 commits) (flutter/engine#8584) (#31098)


https://github.com/flutter/engine/compare/324b8403e8a6...0575a9b46e99

git log 324b8403e8a67db7376f85ed29e0a5792174cca0..0575a9b46e994170ce1290f0025623138411dd13 --no-merges --oneline
0575a9b46 Roll src/third_party/skia bf15b6676843..d51d00745b1d (6 commits) (flutter/engine#8584)

The AutoRoll server is located here: https://autoroll.skia.org/r/flutter-engine-flutter-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff (garyq@google.com), and stop
the roller if necessary.

arbreng committed:
[scenic] Remove dead view_manager ref (#31005)

Robiquet committed:
Added opacity to cupertino switch when disabled (#29451)

aam committed:
Download and handle product version of flutter patched sdk (#31063)

* Support release/debug flavors of flutter_patched_sdk

* Use [anyNamed] instead of [any] for mocking named arguments

* Fix use of local engine in release mode

engine-flutter-autoroll committed:
9970fb550 Roll src/third_party/skia d51d00745b1d..9d10206b7add (1 commits) (flutter/engine#8586) (#31106)


https://github.com/flutter/engine/compare/0575a9b46e99...9970fb550071

git log 0575a9b46e994170ce1290f0025623138411dd13..9970fb5500710471c031cb86f899bfc3f4a6bafd --no-merges --oneline
9970fb550 Roll src/third_party/skia d51d00745b1d..9d10206b7add (1 commits) (flutter/engine#8586)

The AutoRoll server is located here: https://autoroll.skia.org/r/flutter-engine-flutter-autoroll

Documentation for the AutoRoller is here:
https://skia.googlesource.com/buildbot/+/master/autoroll/README.md

If the roll is causing failures, please contact the current sheriff (garyq@google.com), and stop
the roller if necessary.

Commit:
Merge branch 'add_android_embedding_test_project' of github.com:matthew-carroll/flutter into add_android_embedding_test_project

Commit:
Switched to AndroidX to test alarm manager plugin.
